### PR TITLE
Added expiration times for email links; added known issue

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/managing-your-account.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ Google's reCaptcha verification service auto-detects your browser's language.
 
 == Join an Existing Organization
 
-If there is an existing Anypoint Platform organization that you want to join, ask the organization administrator for an invitation. When an administrator invites you to the organization, you receive an email with a link to accept the invitation.
+If there is an existing Anypoint Platform organization that you want to join, ask the organization administrator for an invitation. When an administrator invites you to the organization, you receive an email with a link to accept the invitation. The link in one week.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-your-account.adoc]:
```suggestion
If there is an existing Anypoint Platform organization that you want to join, ask the organization administrator for an invitation. When the administrator responds with an email invitation, you must click the included link within one week.
```

Comment (kbostrom) [reply modules/ROOT/pages/managing-your-account.adoc]: I'm still not too keen on "an Anypoint Platform organization"....

File: modules/ROOT/pages/managing-your-account.adoc [hunk 2]
```diff
@@ -73,15 +73,15 @@ You can use an existing account to join an existing organization once you receiv
 * Admin: An account with the organization administrator role associated to a different organization.
 * Owner: An organization owner account for a different organization.
 
-After clicking the link in the invitation, you can choose the account to use to join the organization.
+After clicking the link in the invitation, you can choose the account to use to join the organization. The link expires in one week.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-your-account.adoc]: Do both the link to accept and the ability to choose an account expire in one week, or just the link to accept? Clarify.
```suggestion
After clicking the link in the invitation, you have one week to choose the account to use to join the organization. 
```

File: modules/ROOT/pages/managing-your-account.adoc [hunk 3]
```diff
@@ -73,15 +73,15 @@ You can use an existing account to join an existing organization once you receiv
...
 
 [NOTE]
 If the account you want to use is already associated with another master organization, an exclamation mark is displayed next to it. This means that additional steps are needed before migrating the account to a new organization.
 
 [[create_new_account]]
 == Create a New Account Within an Existing Organization
 
-You can create a new account within an existing organization only if you receive an email invitation to join the organization.
+You can create a new account within an existing organization only if you receive an email invitation to join the organization. The link expires in one week.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-your-account.adoc]:
```suggestion
You can create a new account within an existing organization only if you receive an email invitation with a link to join the organization. The link expires in one week.
```

File: modules/ROOT/pages/managing-your-account.adoc [hunk 4]
```diff
@@ -175,3 +175,9 @@ You receive an email from Anypoint Platform with a list of user names associated
 == Upgrade Your Account
 
 To upgrade your Anypoint Platform trial account to an enterprise license, contact your account representative or https://www.mulesoft.com/contact[the MuleSoft sales team].
+
+== Known Issue: Account Creation
+
+When you attempt to create an account from an email invitation, an error appears, erroneously stating your credentials are invalid.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-your-account.adoc]:
```suggestion
When you attempt to create an account from an email invitation to join an organization, and you use a username that already exists (even in a different organization), an error appears, erroneously stating your credentials are invalid.
```

File: modules/ROOT/pages/managing-your-account.adoc [hunk 5]
```diff
@@ -175,3 +175,9 @@ You receive an email from Anypoint Platform with a list of user names associated
...
+This error occurs when a username already exists, even if it is under a different organization.
+If the username already belongs to you, you can move the username from your previous organization to the organization that sent you the email invitation.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-your-account.adoc:183]: "If the username belongs to you in another organization, you can move the name to the organization that sent you the email invitation."

What if the username is already being used by someone else?

**Review by kbostrom — APPROVED** [review]
quick and dirty!

File: modules/ROOT/pages/managing-your-account.adoc (hunk 5 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/managing-your-account.adoc:183]: They'd have to pick another. If I'm sarathecoubian in MuleSoft and I click a link to join Salesforce, I would encounter this issue and would need my username to be moved.

**Pull request merged**